cmd/backup-operator/README.md

```diff
@@ -27,9 +24,10 @@ NAME                                    READY     STATUS    RESTARTS   AGE
 etcd-backup-operator-1102130733-hhgt7   1/1       Running   0          3s
 ```
 
-### Create AWS Secret
+### Setup AWS Secret
```

Comment:
keep it to `Create the AWS secret` to be consistent with restore-operator readme?

Reply:
That should be changed too.

Reply:
not sure the difference between `Create AWS Secret`  and `Setup AWS Secret`. it seems same to me in this context.

Reply:
anyways, be consistent is the key here. I am fine with either `Create AWS Secret` or `Setup AWS Secret`.